src/Sylius/Bundle/ResourceBundle/DependencyInjection/SyliusResourceExtension.php

```diff
@@ -46,6 +46,14 @@ public function load(array $config, ContainerBuilder $container)
             $loader->load($configFile);
         }
 
+        $container->setAlias('sylius.resource_controller.authorization_checker', 'sylius.resource_controller.authorization_checker.disabled');
+
+        if (class_exists('Sylius\Bundle\RbacBundle\SyliusRbacBundle')) {
```

Comment:
`::class` still works if the class doesn't exist.
same goes for where you refer to classes in the `getClassMetadataClassname` method in the `Sylius\Bundle\ResourceBundle\DependencyInjection\Driver\Doctrine` namespace